openapps/opencoap/opencoap.c

```diff
@@ -169,15 +171,31 @@ void opencoap_receive(OpenQueueEntry_t* msg) {
 
    // process handled options
    //== Stateless Proxy option
-   statelessProxy = opencoap_find_option(coap_incomingOptions, coap_incomingOptionsLen, COAP_OPTION_NUM_STATELESSPROXY);
+   option_count = opencoap_find_option(coap_incomingOptions, coap_incomingOptionsLen, COAP_OPTION_NUM_STATELESSPROXY, &option_index);
+   if(option_count >= 1)
```

Comment:
We typically use the syntax:

if(condition) { 

} else {

}

with braces on the same line as the if/else condition. Same for other loops. Could you update your PR to reflect this?